yaml/shared/src/main/scala/org/virtuslab/yaml/internal/load/reader/token/FinalBreak.scala

```diff
@@ -0,0 +1,36 @@
+package org.virtuslab.yaml.internal.load.reader.token
+
+enum FinalBreak(indicator: Char):
+  /**
+   * Final break is excluded from content
+   */
+  case Keep extends FinalBreak('+')
+
+  /**
+   * Final break and any trailing empty lines are included to content
+   */
+  case Strip extends FinalBreak('-')
+
+  /**
+    * Final break is included to content, rest of empty lines are excluded. Default behaviour
+   */
+  case Clip extends FinalBreak(' ')
+end FinalBreak
```

Comment:
I would add here both link to the [spec](https://yaml.org/spec/1.2/#id2794534) and some exaples in addition to comments